<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupDialog</class>
 <widget class="QWidget" name="StartupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>126</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Menu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="openMap">
     <property name="text">
      <string>Open Map</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="openCampaign">
     <property name="text">
      <string>Open Campaign</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createMap">
     <property name="text">
      <string>Create Map</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="quit">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
